crates/oxc_transformer/src/typescript/options.rs

```diff
@@ -45,6 +48,11 @@ pub struct TypeScriptOptions {
 
     /// Unused.
     pub optimize_const_enums: bool,
+
+    // Preset options
+    /// Rewrite the import extensions to js extensions
```

Comment:
Can you copy over some of the documentation from https://babel.dev/docs/babel-preset-typescript#rewriteimportextensions?

Reply:
Updated, made some changes